CONTRIBUTORS

```diff
@@ -10,3 +10,4 @@
 #     Name <email address>
 
 Hassan Syed <h.a.syed@gmail.com>
+Jason Feinstein <jwf@google.com>
```

Comment:
I think googlers are by definition assumed to be contributing. (I have no problems with adding explicit names here, but when I was at Google, we never used to include googlers, it was "Google engineers AND these folks" (see Guava for instance, where googlers appear only because of their pre-googler contributions)

Reply:
Ah ok, I wasn't sure given the header for this file...